### PR TITLE
Implement command line arguments

### LINKED ISSUE #6: Implement command line arguments
Arguments:
- [x] Help
- [x] Debug
- [x] Verbose
- [x] Version

See: https://gitlab.com/oxr463/check-generator/-/blob/master/src/generator.sh#L14

### PR DESCRIPTION
**Pull request by outzhu**
testing the Pull Request feature and adding new code to PII.

Bug: https://github.com/infinite-omicron/pii-redactor/issues/6

### CODE REVIEW

File: pii_redactor.sh [hunk 1]
```diff
@@ -1,8 +1,39 @@
-#!/bin/bash  
+#!/bin/bash
 
+#varibles in use
```
Comment (oxr463) [inline pii_redactor.sh]:
```suggestion
# variables in use
```

**Review by oxr463 — CHANGES_REQUESTED** [review]

**Review by oxr463 — APPROVED** [review]

**Comment by oxr463** [comment]
Since this is still WIP, I'm going to convert it to a draft.

File: pii_redactor.sh [hunk 2]
```diff
@@ -1,8 +1,39 @@
...
+EOF
+    ;;
+  -d|--debug)
+    debug=1
+    ;;
+  -v|--verbose)
+    verbose=1
+    ;;
+  -ver|--version)
+    echo "PII_Redaction.sh version is ${version}"
```
Comment (oxr463) [inline pii_redactor.sh]:
```suggestion
    echo "pii-redactor version ${version}"
```

**Review by outzhu — COMMENTED** [review]
adding changes to main res

**Review by outzhu — COMMENTED** [review]
upgrade

File: pii_redactor.sh [hunk 3]
```diff
@@ -1,18 +1,33 @@
 #!/bin/bash
 set -eu
 # variables in use
-version="0.0.3"
+version="0.0.4"
```
Comment (oxr463) [inline pii_redactor.sh]: I wouldn't increment the version number until we actually release the first version.
```suggestion
version="0.0.1"
```

File: .github/workflows/checker.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: Shellchecker
```
Comment (oxr463) [inline .github/workflows/checker.yml]:
```suggestion
name: shellcheck
```

**Comment by oxr463** [comment]
1. No arguments.

```
./pii_redactor.sh 
./pii_redactor.sh: line 9: $1: unbound variable
```

2. Current directory:

```
./pii_redactor.sh .
Success
```

3. Help flag:

```
./pii_redactor.sh -h
USAGE: 
pii_redactor [command] || <directory that holds cluster.yml>

GLOBAL COMMANDS:
  -h, --help            Show this message
  -d, --debug           Output debug messages
  -v, --verbose         verbose mode
  -ver, --version       Print version
```

**Pull request merged**